mlir/tools/mlir-miopen-driver/mlir-miopen-driver.cpp

```diff
@@ -50,11 +50,17 @@ static cl::opt<std::string> outputFilename("o", cl::desc("Output filename"),
                                            cl::value_desc("filename"),
                                            cl::init("-"));
 
-static cl::opt<std::string>
-    operation("operation",
-              cl::desc("Convolution operation, eg: conv2d, conv2d_bwd_data, "
-                       "conv2d_bwd_weight..."),
-              cl::value_desc("convolution flavor string"), cl::init("conv2d"));
+static cl::opt<mlir::Conv2dGenerator::Op> operation(
+    "operation", cl::desc("Convolution operation,"),
+    cl::values(clEnumValN(Conv2dGenerator::Op::Fwd, "conv2d",
+                          "Forward convolution"),
+               clEnumValN(Conv2dGenerator::Op::Dummy, "conv2d_dummy",
+                          "Pretend to perform a forward convolution"),
```

Comment:
I believe we shouldn't allow user to populate a conv2d dummy op

Reply:
Actually maybe conv2d dummy op shouldn't be used anywhere at all, though I'm not sure. @whchung: Are we still using the conv2d dummy op?